src/main/scala/com/gu/inspectorlambda/aws/AWSInspector.scala

```diff
@@ -90,6 +105,29 @@ class AWSInspector(val client: AmazonInspector) extends StrictLogging {
     createAssessmentTargetResult.getAssessmentTargetArn
   }
 
+  // These implementations can be deleted once the old bad targets and
+  // templates have been removed.
+  def deleteOldAssessmentTemplate(name: String): Unit = {
+    val assessmentTemplateArns = client.listAssessmentTemplates(new ListAssessmentTemplatesRequest()).getAssessmentTemplateArns
+    if (assessmentTemplateArns.isEmpty)
+      None
+    else {
+
+      val matchingAssessmentTemplates = client.describeAssessmentTemplates(new DescribeAssessmentTemplatesRequest().withAssessmentTemplateArns(assessmentTemplateArns)).getAssessmentTemplates.asScala
+        .filter(assessmentTemplate => assessmentTemplate.getName.equals(name))
+
+      // delete if arn is not correct
+      matchingAssessmentTemplates
+        .foreach(assessmentTemplate => {
+          val deleteAssessmentTemplateRequest = new DeleteAssessmentTemplateRequest()
+            .withAssessmentTemplateArn(assessmentTemplate.getArn)
+          client.deleteAssessmentTemplate(deleteAssessmentTemplateRequest)
+          Thread.sleep(3)
```

Comment:
See above. Is a 3 millisecond pause valuable?

Reply:
That was supposed to be 3 secs!  Now I'm not convinced it is needed after all.